### PR TITLE
Read the genetics portal link base URL from config

### PR DESCRIPTION
**Pull request by fedde-s**
To allow alternative installations of the platform to link to
corresponding alternative installations of the genetics portal, for all
genetics portal links corresponding to the current page or row of data.
Similar to the setting for the API URL (and the one for the EFO URL).

### CODE REVIEW

**Comment by fedde-s** [comment]
I originally made this option apply to the genetic variant links in the **PheWAS Catalogue** table as well, but of course that no longer applies :smile:  Hence the force-push immediately after creating the pull request.

**Comment by d0choa** [comment]
@carcruz and @mbdebian make sure you are both happy on your respective sides, so we can merge

**Comment by fedde-s** [comment]
Don't hesitate to let me know if there's a downside I missed! I thought this would be useful to others too, perhaps even to you if testing/previewing links to genetics portal data that does not (yet) appear in the public instance.

**Comment by fedde-s** [comment]
Rebased to resolve the conflicts with <https://github.com/opentargets/platform-app/pull/543>.

File: public/config.template [hunk 1]
```diff
@@ -1,21 +1,6 @@
...
-var configPartnerTargetSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_TARGET_SECTION_IDS
-var configHideDrugSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HIDE_DRUG_SECTION_IDS
-var configPartnerDrugSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_DRUG_SECTION_IDS
-var configHideEvidenceSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HIDE_EVIDENCE_SECTION_IDS
-var configPartnerEvidenceSectionIds = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_EVIDENCE_SECTION_IDS
-var configPartnerColorRange = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_COLOR_RANGE
-var configHideDataTypes = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_HIDE_DATA_TYPES
-var configPartnerDataTypes = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_DATA_TYPES
-var configPartnerDataSources = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_PARTNER_DATA_SOURCES
-var configDownloadsURL = DEVOPS_CONTEXT_PLATFORM_APP_CONFIG_DOWNLOADS_URL
```
Comment (carcruz) [inline public/config.template:21]: Hello @fedde-s, thanks for your PR, it looks good. Only, these variables are needed for our build/release

I'll need to do a quick look at it with the backend team in order to add the variable to our release process.

**Review by carcruz — CHANGES_REQUESTED** [review]

File: public/config.template (hunk 1 above)
Comment (fedde-s) [reply public/config.template:21]: Are you sure? As I explained in the commit message, the code very much suggests that none of these variables do anything anymore as of the change I linked, and the build process already uses the replacement for them.

Comment (fedde-s) [reply public/config.template:21]: Ah, I do see an oversight in my commit: the broken (and now cleaned-up) variables are still listed in `docs/CUSTOM_CONFIG.md`.

Comment (fedde-s) [reply public/config.template:21]: For completeness: the patch that removed the logic of reading these variables is commit <https://github.com/opentargets/platform-app/commit/2f40f839e94faffefc12aa5f35a2b71b86d46fb7> in pull request <https://github.com/opentargets/platform-app/pull/487>

Comment (fedde-s) [reply public/config.template:21]: For completeness, the changes that remove the logic to read the variables (and adds the new way to set it, already used in the Terraform config, see my commit message) are part of pull request <https://github.com/opentargets/platform-app/pull/487>.

Comment (carcruz) [reply public/config.template:21]: Thanks, @fedde-s. I review it with the team, and your right.

**Review by carcruz — APPROVED** [review]

**Pull request merged**